docker/README.md

```diff
@@ -9,10 +9,52 @@ You will need to install Docker to be able to run any of the following scripts.
 Make sure you have [Docker](https://docker.com) installed and configured by
 running `docker -v`.
 
-For HPI VMs, [this
-script](https://gist.github.com/johannesvedder/0fafcbeabe8e069f96085bfedaebd9d0)
+For HPI VMs,
+[this script](https://gist.github.com/johannesvedder/0fafcbeabe8e069f96085bfedaebd9d0)
 can be used to install and setup docker.
 
+All the following commands are meant to be run from this directory, i.e. the
+`docker/` directory of the root repository, unless otherwise specified.
+
+## Quick Start with the StudyU CLI
+
+To get started quickly, run the StudyU CLI. It will interactively guide you
+through the setup process. The CLI is a wrapper around the docker-compose
+commands described in the following sections. It is located in the root
+directory of the repository and can be run with `./studyu-cli.sh`.
+
+During the configuration of the CLI, you will be asked to define the components
+you want to run. The following components are available:
+
+- `supabase-db`: A PostgreSQL database for Supabase
+- `supabase`: A self-hosted Supabase instance (needs `supabase-db`)
+- `proxy`: Nginx reverse proxy to access Supabase if no StudyU component is running
+- `studyu-app`: The StudyU App (includes `proxy`)
+- `studyu-designer`: The StudyU Designer (includes `proxy`)
+
+A minimal full setup would be `supabase-db`, `supabase`, `studyu-app` and
+`studyu-designer`. If you only want to run a self-hosted Supabase
+instance and deploy StudyU elsewhere, you can choose `proxy` instead of
+`studyu-app` and `studyu-designer` to access Supabase Studio. Otherwise, if you
+only want to run the StudyU App and the StudyU Designer and not self-host Supabase,
+you can choose the respective component without `supabase-db` and `supabase`.
+Have a look into the use cases below for more details on the different
+configurations.
+
+```bash
+chmod +x studyu-cli.sh
```

Comment:
Why is this necessary? Can't we just commit the `+x` mode to the repo?

Reply:
Don't know what the best-practise is here, but if there are no downsides, absolutely.

Reply:
I always do it like this and haven't come across any downsides thus far🙈